guiType.py

```diff
@@ -23,7 +23,7 @@
 root.geometry("1024x768")  # 1024x768 # set resolution
 glblBGC = "#343d46"
 root.config(bg=glblBGC)
-if platformsystem() != "Windows" and platformsystem() != "Darwin":
+if platform.system() != "Windows" and platform.system() != "Darwin":
```

Comment:
FYI.  This line and those above have nothing to do with fixing bugs.

As I review this PR, I try to understand how each line fixes a bug.  The details of the bug isn't mentioned, so I don't know the symptoms.  I have to guess what the symptoms may be, apply logic and some trial & error to validate my guess, and revise before I have any sense of how changing this line fixed a bug.  This takes time to do, and it's time saved for the reviewer if these unrelated changes were just thrown in a small commit titled "Refactor/cleanup".   Then my only focus on that cleanup commit is to verify no behavior is changed in the program and move on to the bug fix commit.